src/blueapi/service/main.py

```diff
@@ -313,7 +314,15 @@ def set_active_task(
         raise HTTPException(
             status_code=status.HTTP_409_CONFLICT, detail="Worker already active"
         )
-    runner.run(interface.begin_task, task)
+    runner.run(
+        interface.begin_task,
+        task=task,
+        pass_through_headers={
+            key: request.headers[key]
+            for key in {"Authorization"}
+            if key in request.headers
+        },
```

Comment:
```suggestion
        pass_through_headers={
            key: value
            for key, value in request.headers.items()
            if key in {"Authorization"}
        },
```
nit, invert this: presumably later more headers will be needed?

Reply:
Done.